packages/server/graphql/public/typeDefs/addFeatureFlagToOrg.graphql

```diff
@@ -0,0 +1,41 @@
+"""
+A flag to give an individual organization super powers
+"""
+enum OrganizationFlagEnum {
+  teamsLimit
+}
+
+"""
+The types of flags that give an individual organization super powers
+"""
+type OrganizationFeatureFlags {
+  teamsLimit: Boolean!
+}
+
+extend type Mutation {
+  """
+  Give some organizations advanced features in a flag
+  """
+  addFeatureFlagToOrg(
+    """
+    a list of organization ids
+    """
+    orgIds: [String!]!
+
+    """
+    the flag that you want to give to the organization
+    """
+    flag: OrganizationFlagEnum!
+  ): AddFeatureFlagToOrgPayload!
+}
+
+type AddFeatureFlagToOrgPayload {
+  error: StandardMutationError
```

Comment:
+1 we have a newer pattern where we return a success object and a union or an error payload. That way, we know that if it's not an error payload, `organizations` definitely exist. See example [here](https://github.com/ParabolInc/parabol/blob/13cabc3c6bd6b2082c2d7e37a46695a1498f6221/packages/server/graphql/private/typeDefs/changeEmailDomain.graphql)